README.md

```diff
@@ -6,14 +6,24 @@
 
 The product selector extension allows content authors to easily search and select products in Salesforce Commerce Cloud and add them to your content.
 
-The IDs of each product selected are added to the content as an array of strings.
+The IDs of each product selected are added to the content as an array of strings for SFCC and an array of objects with the id and catalog for Hybris.
```

Comment:
I'm wondering if we should have two different sections for configuration, one for SFCC and one for Hybris. Just think it gets confusing mixing the two and especially if we add commerce cloud becuase that will likely have different params again.